packages/components/src/Form/Inputs/InputFilters/InputFilters.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import styled from 'styled-components'
+import React, { FC, useState, useRef, MouseEvent } from 'react'
+import { Select } from '../Select'
+import { InputText } from '../InputText'
+import { Icon } from '../../../Icon'
+import { IconButton } from '../../../Button'
+import { Chip } from '../../../Chip'
+import { Space } from '../../../Layout'
+
+export interface AvailableFilter {
+  value: string
+}
+
+export interface InputFiltersProps {
+  className?: string
+  availableFilters: AvailableFilter[]
+  fieldFilters?: string[]
+  hideFilterIcon?: boolean
+}
```

Comment:
These should get annotated with JS Doc strings. E.g.:

```
/**
  * List of filters the user can select from
  **/
availableFilters: ...
```

I also think the typing should change a bit:

```suggestion
export interface FieldFilter {
   /* specify field to filter against */
   field: string
   /* text displayed in drop-down, optional, `field` is used if not specified */
   label?: string
}

export interface FieldFilterExpressions extends FieldFilter {
   expression?: string
}

export interface InputFiltersProps {
  available: FieldFilter[]
  filters?: FieldFilterExpressions[]
  hideFilterIcon?: boolean
  className?: string
}
```